specification/network/resource-manager/Microsoft.Network/preview/2021-02-01-preview/examples/NetworkManagerAdminRulesListByConfig.json

```diff
@@ -0,0 +1,58 @@
+{
+  "parameters": {
+    "api-version": "2021-02-01-preview",
+    "subscriptionId": "subId",
+    "resourceGroupName": "rg1",
+    "networkManagerName": "testNetworkManager",
+    "configurationName": "myTestSecurityConfig"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "id": "/subscriptions/subId/resourceGroups/rg1/providers/Microsoft.Network/networkmanagers/testNetworkManager/securityAdminConfigurations/myTestSecurityConfig/ruleCollections/testRuleCollection/rules/SampleAdminRule",
+            "type": "Microsoft.Network/networkManagers/securityAdminConfigurations/ruleCollections/rules",
```

Comment:
GET collection of a grand child is not a valid scenario. 
RPC live validations check requested resources and the type returned are same.

Refer
[https://armwiki.azurewebsites.net/api_contracts/RPCstrongvalidation.html](https://armwiki.azurewebsites.net/api_contracts/RPCstrongvalidation.html)

[https://portal.azure-devex-tools.com/tools/validation/menu](https://portal.azure-devex-tools.com/tools/validation/menu)